scripts/create-package/plop-templates-react/package.json.hbs

```diff
@@ -28,6 +28,7 @@
     "@fluentui/eslint-plugin": "",
     "@fluentui/jest-serializer-make-styles": "",
     "@fluentui/react-conformance": "",
+    "@fluentui/react-conformance-make-styles": "",
```

Comment:
Can you add the new tests to `scripts/create-component/plop-templates-component/src/components/{{componentName}}/{{componentName}}.test.tsx.hbs` too?

Reply:
Yep, good call, done 👍

Reply:
From what I can see they were added to `isConformant` which is what you were looking for